platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -182,8 +182,8 @@ private void wire() {
 
         solderNonAncientEventWindow();
         linkedEventIntakeWiring
-                .minimumGenerationNonAncientOutput()
-                .solderTo(pcesWriterWiring.minimumGenerationNonAncientInput(), INJECT);
+                .nonAncientEventWindowOutput()
```

Comment:
This call should be bundled into `solderNonAncientEventWindow()`

Reply:
Agree, fixed.